recipes/croaring/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "CRoaring" %}
+{% set version = "0.2.59" %}
+{% set sha256 = "02f80850e856001dbcd2571d140bab3d2bfb1e5bcc1e3b271d4c4e964dd65518" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RoaringBitmap/CRoaring/archive/v0.2.59.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}    
+    - cmake
+  run:
+    - gcc_linux-64    # [linux]
+    - clang_osx-64    # [osx]
```

Comment:
Do you need the compilers at runtime? If so, you will have to delete this part and use as ``{{ compiler('cxx')}}``

Reply:
Actually, I don't think I need to specify the `run`